imagersite_project/imager_images/models.py

```diff
@@ -0,0 +1,46 @@
+# Imager_images model
+
+from __future__ import unicode_literals
+from django.db import models
+from django.dispatch import receiver
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+import uuid
+
+
+class Photo(models.Model):
+"""Photos submitted by user."""
+    photo_id = models.UUIDField(
+        primary_key=True,
+        default=uuid.uuid4,
+        editable=False)
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    albums = models.ManyToManyField('Album')
+    file = models.ImageField(upload_to=user_directory_path)
+    date_uploaded = models.DateTimeField('date uploaded', blank=True)
+    date_modified = models.DateTimeField('date modified', blank=True)
+    date_published = models.DateTimeField('date published', blank=True)
+    published = models.CharField(max_length=128,
```

Comment:
If you're specifying what choices you want for this field, why have a `max_length` of 128? Your longest word here is "private", which is 7 characters long. Why not just truncate at 10 or 20 (in case you want some other slightly longer "published" status)?